db/schema.rb

```diff
@@ -10,15 +10,14 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_07_03_012614) do
-
+ActiveRecord::Schema[8.0].define(version: 2025_06_27_212857) do
   # These are extensions that must be enabled in order to support this database
-  enable_extension "plpgsql"
+  enable_extension "pg_catalog.plpgsql"
 
   create_table "blocked_referrers", force: :cascade do |t|
     t.string "host_with_path", null: false
-    t.datetime "created_at", precision: 6, null: false
```

Comment:
Do you know why the schema changed these things (besides the `last_seen_at`)? Is it just part of it being an `[8.0]` schema now?

Reply:
Seems like it: https://stackoverflow.com/a/71482301/638966

This is the first migration in 5 years so it makes sense we've got some changes.